### ISSUE TITLE
Cannot Define Key Binding In @command Node

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#4090]
It used to be that a key binding could be defined in an `@command` node in an @settings tree:

`@command my-special-command @key = Ctrl+F6`

This worked for an outline-specific `@settings` tree as well as a global one. I just noticed that this no longer creates a binding in an outline where it had always worked in the past.  I'm not sure when this change happened but I think it was very recent or I would have noticed it before now.

It's still possible to set up the outline-specific bindings in a local `@shortcuts` body but this shouldn't be necessary.   I think the behavior should return to the way it was.

```
Leo 6.8.2-devel, devel branch, build 8bd9c03e99
2024-10-10 12:36:29 -0500
Python 3.12.3, PyQt version 6.7.2
Windows 10 AMD64 (build 10.0.19045) SP0
```

**Comment by edreamleo** [comment]
@tbpassin I am the likely culprit. Let me look into this.

**Comment by edreamleo** [comment]
@tbpassin PR #4088 caused this failure. I misunderstood the constraints on `k.registerCommand`. It's docstring says
"Only k.bindOpenWith and the mod_scripting.py plugin should set allowBinding", but the PR mistakenly prohibited the `allowBinding` kwarg *everywhere.

**Comment by tbpassin** [comment]
That was quick!

**Comment by edreamleo** [comment]
Closed via PR #4092.
